test/runtests.jl

```diff
@@ -57,6 +57,50 @@ end
     end
 end
 
+@testset "wigner3j: test orthogonality relations" begin
+    # j1 <= 200 and j2 <= 200 so that this test runs quickly
+    j1 = convert(BigFloat,rand(0:1//2:200))
+    j2 = convert(BigFloat,rand(0:1//2:200))
+    m1 = rand(-j1:j1)
```

Comment:
There is no need to convert j1 and j2 to `BigFloat` in order to have `BigFloat` precision in the calculation of the wigner symbols. Also, rather than using the custom range `0:1//2:200` I would suggest to either use `largejlist` from the top of the file if you just want to select one random element, as you do in this test, or `smalljlist` if you want to run over all values. You can certainly suggest to change the limits of those two constants. 

However, rather than picking random j's and m's, I would suggest to add loops over small ranges of `j` (`smalljlist`) and compatible values of other `j`s, `l`s and `m`s. The tests use a fixed seed into `Random`, meaning that every CI run will run exactly the same values of `j`.